backend/src/main/java/com/bakdata/conquery/sql/conversion/dialect/HanaSqlDialect.java

```diff
@@ -30,6 +32,11 @@ public DSLContext getDSLContext() {
 		return this.dslContext;
 	}
 
+	@Override
+	public SqlCDateSetParser getCDateSetParser() {
+		return new DefaultSqlCDateSetParser();
```

Comment:
In der PostgresSqlDialect Klasse ist dieser Parser ein Member und wird nicht bei jedem Call neuerzeugt. Hat das einen Grund?

Reply:
Nein, war nur ein Versehen. Danke!